03_letters/madhavansu/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	s    string
+	lmap map[rune]int
+	ss   []string
+}{
+
+	{"pwopoip@oiopoi@",
+		map[rune]int{'o': 5, 'p': 4, 'i': 3, '@': 2, 'w': 1},
+		[]string{"@:2", "i:3", "o:5", "p:4", "w:1"}},
+	{"‣£ê‣££~ 3 3 3",
+		map[rune]int{'3': 3, '£': 3, ' ': 3, '‣': 2, 'ê': 1, '~': 1},
+		[]string{" :3", "3:3", "~:1", "£:3", "ê:1", "‣:2"}},
+}
```

Comment:
Would like to see a test case against an empty input string `""`.

Reply:
Updated.